utils/apollo/schema/resolvers.js

```diff
@@ -1,54 +1,130 @@
-import dbConnect from 'utils/dbConnect';
 import User from 'models/User';
```

Comment:
as you start to use more resolvers, can you separate them into separate files? Ex. create a folder `resolvers` and put each resolver into its own file.

Reply:
Should we split up the resolvers into Query or Mutation folder, or one file for each resolver?

Reply:
maybe one folder for queries and one folder for mutations. Then we can separate each resolver into their own file under their respective folder. Finally, we can export all the resolves in query under an index.js in the query folder and the same thing with the mutation folder and then rebuild our main resolver elsewhere by importing the two index.js files. If you're confused about the index.js thing, reference this file for a somewhat-similar example of what I mean (example is with redux though): https://github.com/LemonFace0309/Burger-Builder/blob/master/src/store/actions/index.js